PicaGBV.js

```diff
@@ -345,9 +345,13 @@ function doExport() {
 		//Sonstige Anmerkungen (manuell eintragen) --> 4201
 		writeLine("4201", "");
 		
-		//Inhaltliche Zusammenfassung -->4207
+		//Inhaltliche Zusammenfassung --> 4207/4209
 		if (item.abstractNote && exportAbstract) {
```

Comment:
I added here the check for the variable `exportAbstract` again. It is defined at the very top of this file. However, we could also add an option in the export dialog for that.